components/global_header/left_controls/product_menu/product_menu.tsx

```diff
@@ -80,10 +80,16 @@ const ProductMenu = (): JSX.Element => {
     const exploreToolsTourTriggered = triggerStep === GenericTaskSteps.STARTED;
 
     const pluginsList = useSelector((state: GlobalState) => state.plugins.plugins);
-    const boards = pluginsList.focalboard;
     const playbooks = pluginsList.playbooks;
 
-    const showBoardsTour = enableTutorial && tutorialStep === ExploreOtherToolsTourSteps.BOARDS_TOUR && exploreToolsTourTriggered && boards;
+    const boardsPluginEnabled = pluginsList.focalboard;
+    let boardsProductEnabled = false;
+    if (products) {
+        boardsProductEnabled = products.some((product) => product.pluginId === suitePluginIds.focalboard || product.pluginId === suitePluginIds.boards);
+    }
+    const boardsEnabled = boardsPluginEnabled || boardsProductEnabled;
```

Comment:
Could we reuse your new `useGetPluginsActivationState` here?

Reply:
thanks, done